src/hydration.js

```diff
@@ -0,0 +1,36 @@
+class Hydration {
+  constructor(hydrationData, id) {
+    this.data = hydrationData;
+    this.id = id;
+    this.user = this.findUserHydroData();
+  };
+
+  findUserHydroData() {
+    return this.data.filter(user => user.userID === this.id)
+
+  }
+
+  calculateAvgOzsPerDay() {
+    let totalOz = this.user.reduce((total, day) => {
+        total += day.numOunces
+        return total
+      },0);
+    return Math.floor(totalOz/this.user.length)
+
+  };
+
+  calculateDailyOz(date) {
```

Comment:
This method calculates how many ozs of water a user has consumed based on the date parameter that is passed in as an argument

Reply:
Yep, for now, this is hardcoded, not sure if later we will need to do this dynamically.